Controller/Controller.php

```diff
@@ -109,7 +119,7 @@ public function getTranslationsAction(Request $request, $domain, $_format)
         $locales = $this->getLocales($request);
 
         if (0 === count($locales)) {
-            throw new NotFoundHttpException();
+            return (new Response())->setStatusCode(Response::HTTP_NOT_FOUND);
```

Comment:
This should be reverted. What blocker do you have ?

Reply:
`ControllerTest.php` `testGetTranslationsWithPathTraversalAttack` and `testGetTranslationsWithLocaleInjection` expect a Response with status 404, but get a NotFoundException. 

This works fine in Symfony ^4.0 and ^5.0, but in Symfony 3.4 this fails (probably because the 3.4 exception handler does not turn it into a 404 Response object, whereas the 4.0 and 5.0 handlers do).

I thought it made more sense to return an actual 404 response, since according to the `ControllerTest` that is what we expect to get anyway (line 184 and 201 of `ControllerTest.php`).

I would say either return an actual 404 response 
**OR** change the ControllerTest to test for an uncaught `NotFoundException` instead of 404 response

If you want to reproduce the error:
- Revert these changes
- Symfony 3.4 on PHP 7.2 -> run tests

Reply:
For now I think it is ok. Symfony 3.4 will be a pain to support always so when it gets deprecated on November 2020 we can change this piece of code.